.gitignore

```diff
@@ -56,6 +56,7 @@ target/
 
 # Local utility scripts
 analyze.py
+/out/
```

Comment:
This feels like we're working around the problem rather than fixing the problem. Wouldn't a better solution be to have `create_baseline_stubs.py` do a better job at cleaning up after the stubs have been generated?

You can change stubgen's output directory by using the `--output` command-line option. I think a better solution to this problem would be to change `create_baseline_stubs.py` so that stubgen outputs the generated stubs directly into `stub_dir`, instead of outputting them into an `out` directory and then copying them across from `out` to `stub_dir`.

(I'm happy to do that fix in a separate PR if you want to limit this PR to just the `pyrightconfig.stricter.json` fix :)

Reply:
It was an easy enough change